GHzDACs/ghz_fpga_server.py

```diff
@@ -859,9 +863,15 @@ def timeoutReport(self, runners, results):
         """Create a nice error message explaining which boards timed out."""
         lines = ['Some boards failed:']
         for runner, (success, result) in zip(runners, results):
+            if runner.executionCount is None:
+                executionCount = 'not available'
+            else:
+                executionCount = runner.executionCount
```

Comment:
I see this was already merged, but for future reference this could be written as
```python
executionCount = getattr(runner, 'executionCount', 'not available')
```